data_diff/utils.py

```diff
@@ -188,3 +191,21 @@ def columns_type_changed_template(columns_type_changed) -> str:
 
 def no_differences_template() -> str:
     return "[bold][green]No row differences[/][/]\n"
+
+
+def print_version_info() -> None:
+    base_version_string = f"Running with data-diff={__version__}"
+    logger = getLogger(__name__)
+    latest_version = None
+    try:
+        response = requests.get(url="https://pypi.org/pypi/data-diff/json", timeout=3)
+        response.raise_for_status()
+        response_json = response.json()
+        latest_version = response_json["info"]["version"]
+    except Exception as ex:
+        logger.debug(f"Failed checking version: {ex}")
+
+    if latest_version and parse_version(__version__) < parse_version(latest_version):
+        print(f"{base_version_string} (Update {latest_version} is available!)")
```

Comment:
Not a big fan of this wording, but trying to be succinct

Reply:
@leoebfolsom curious if you have thoughts

Reply:
For reference:
```
dbt-analytics [contact-history-stg] dbt --version
Core:
  - installed: 1.3.4
  - latest:    1.5.0 - Update available!

  Your version of dbt-core is out of date!
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation

Plugins:
  - snowflake: 1.3.0 - Update available!

  At least one plugin is out of date or incompatible with dbt-core.
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation
  ```
  
  borrowing dbt's style isn't usually a bad thing

Reply:
```
dbt-analytics [contact-history-stg] dbt run -s stg_salesforce__contact_history
22:54:49  Running with dbt=1.3.4
22:54:50  Found 140 models, 173 tests, 3 snapshots, 0 analyses, 435 macros, 0 operations, 7 seed files, 81 sources, 0 exposures, 0 metrics
```

Reply:
maybe: 
```
running with data-diff=0.7.7 (latest version:  1.5.0 - Update available!)
```

I think what you have is fine, just throwing stuff out. I'll back away now and let Leo chime in.

Reply:
I would do this:

```
Running with data-diff v0.7.7.  data-diff v0.8.0 is available!

To update, run: pip install data-diff==0.8.0
```